Upgrade of empty instance does not create default namespace table mapping nodes

**Describe the bug**
Upgraded a 2.1.4 instance with no tables in the default namespace. Tried to create the `ci` table in the 4.0 instance and the create table fate operation failed because the zookeeper node for the default namespace table mapping did not exist.

```
Caused by: org.apache.zookeeper.KeeperException$NoNodeException: KeeperErrorCode = NoNode for /namespaces/+default/tables
        at org.apache.zookeeper.KeeperException.create(KeeperException.java:118) ~[zookeeper-3.8.0.jar:3.8.0]
        at org.apache.zookeeper.KeeperException.create(KeeperException.java:54) ~[zookeeper-3.8.0.jar:3.8.0]
        at org.apache.zookeeper.ZooKeeper.getData(ZooKeeper.java:1972) ~[zookeeper-3.8.0.jar:3.8.0]
        at org.apache.accumulo.core.zookeeper.ZooSession.getData(ZooSession.java:296) ~[accumulo-core-4.0.0-SNAPSHOT.jar:4.0.0-SNAPSHOT]
        at org.apache.accumulo.core.fate.zookeeper.ZooReaderWriter.lambda$mutateExisting$7(ZooReaderWriter.java:198) ~[accumulo-core-4.0.0-SNAPSHOT.jar:4.0.0-SNAPSHOT]
        at org.apache.accumulo.core.fate.zookeeper.ZooReader.retryLoopMutator(ZooReader.java:174) ~[accumulo-core-4.0.0-SNAPSHOT.jar:4.0.0-SNAPSHOT]
        at org.apache.accumulo.core.fate.zookeeper.ZooReaderWriter.mutateExisting(ZooReaderWriter.java:196) ~[accumulo-core-4.0.0-SNAPSHOT.jar:4.0.0-SNAPSHOT]
        at org.apache.accumulo.core.util.tables.TableMapping.put(TableMapping.java:70) ~[accumulo-core-4.0.0-SNAPSHOT.jar:4.0.0-SNAPSHOT]
        at org.apache.accumulo.manager.tableOps.create.PopulateZookeeper.call(PopulateZookeeper.java:58) ~[accumulo-manager-4.0.0-SNAPSHOT.jar:4.0.0-SNAPSHOT]
        at org.apache.accumulo.manager.tableOps.create.PopulateZookeeper.call(PopulateZookeeper.java:34) ~[accumulo-manager-4.0.0-SNAPSHOT.jar:4.0.0-SNAPSHOT]
        at org.apache.accumulo.manager.tableOps.TraceRepo.call(TraceRepo.java:76) ~[accumulo-manager-4.0.0-SNAPSHOT.jar:4.0.0-SNAPSHOT]
        at org.apache.accumulo.core.fate.FateExecutor.executeCall(FateExecutor.java:546) ~[accumulo-core-4.0.0-SNAPSHOT.jar:4.0.0-SNAPSHOT]
        at org.apache.accumulo.core.fate.FateExecutor$TransactionRunner.execute(FateExecutor.java:439) ~[accumulo-core-4.0.0-SNAPSHOT.jar:4.0.0-SNAPSHOT]
        at org.apache.accumulo.core.fate.FateExecutor$TransactionRunner.run(FateExecutor.java:379) ~[accumulo-core-4.0.0-SNAPSHOT.jar:4.0.0-SNAPSHOT]
```

**Versions (OS, Maven, Java, and others, as appropriate):**
 - Affected version(s) of this project: 4.0.0

**To Reproduce**
Create a 2.1.4 instance
Upgrade to 4.0.0
Log in and create a table in the default namespace

**Expected behavior**
Table should be created

**Additional context**
This may not be a bug per se, as it's unlikely that somone would try to upgrade a new empty instance.

--- a/server/manager/src/main/java/org/apache/accumulo/manager/upgrade/Upgrader12to13.java
+++ b/server/manager/src/main/java/org/apache/accumulo/manager/upgrade/Upgrader12to13.java
@@ -421,6 +421,9 @@ public class Upgrader12to13 implements Upgrader {
               return tableName;
             });
       }
+      // Ensure the default namespace table mapping node gets created
+      // in case there are not tables in the default namespace
+      mapOfTableMaps.putIfAbsent(Namespace.DEFAULT.id().canonical(), new HashMap<>());
       for (Map.Entry<String,Map<String,String>> entry : mapOfTableMaps.entrySet()) {
         zrw.putPersistentData(Constants.ZNAMESPACES + "/" + entry.getKey() + Constants.ZTABLES,
             NamespaceMapping.serializeMap(entry.getValue()), ZooUtil.NodeExistsPolicy.FAIL);

--- a/server/manager/src/test/java/org/apache/accumulo/manager/upgrade/Upgrader12to13Test.java
+++ b/server/manager/src/test/java/org/apache/accumulo/manager/upgrade/Upgrader12to13Test.java
@@ -38,6 +38,7 @@ import java.util.UUID;
 
 import org.apache.accumulo.core.Constants;
 import org.apache.accumulo.core.client.NamespaceNotFoundException;
+import org.apache.accumulo.core.clientImpl.Namespace;
 import org.apache.accumulo.core.clientImpl.NamespaceMapping;
 import org.apache.accumulo.core.data.InstanceId;
 import org.apache.accumulo.core.fate.zookeeper.ZooReader;
@@ -117,6 +118,7 @@ public class Upgrader12to13Test {
           .put(tableId, entry.getValue());
     }
 
+    expectedNamespaceMaps.put(Namespace.DEFAULT.id().canonical(), new HashMap<>());
     for (Map.Entry<String,Map<String,String>> entry : expectedNamespaceMaps.entrySet()) {
       expect(zrw.putPersistentData(
           eq(Constants.ZNAMESPACES + "/" + entry.getKey() + Constants.ZTABLES),

